bundles.go

```diff
@@ -127,15 +128,49 @@ func NewBundle() (bundle *Bundle, err error) {
 }
 
 func GenerateBundle(bundle *Bundle) (bool, error) {
-	args := []string{
-		"--add-section", fmt.Sprintf(".osrel=%s", bundle.OSRelease), "--change-section-vma", ".osrel=0x20000",
-		"--add-section", fmt.Sprintf(".cmdline=%s", bundle.Cmdline), "--change-section-vma", ".cmdline=0x30000",
-		"--add-section", fmt.Sprintf(".linux=%s", bundle.KernelImage), "--change-section-vma", ".linux=0x2000000",
-		"--add-section", fmt.Sprintf(".initrd=%s", bundle.Initramfs), "--change-section-vma", ".initrd=0x3000000",
+	type section struct {
+		section string
+		file    string
+	}
+	sections := []section{
+		{".osrel", bundle.OSRelease},
+		{".cmdline", bundle.Cmdline},
+		{".splash", bundle.Splash},
+		{".linux", bundle.KernelImage},
+		{".initrd", bundle.Initramfs},
+	}
+
+	var args []string
+	vma, err := func() (int64, error) {
```

Comment:
@Foxboron 

systemd's `ukify` rounds up to block size (4096). Should we do the same?

https://github.com/systemd/systemd/blob/d09df6b94e0c4924ea7064c79ab0441f5aff469b/src/ukify/ukify.py#L80-L90

Reply:
Unclear to me why. But yes please.

I also think that needs to be done in `mkinitcpio` when I think about it.

Reply:
Also, `ukify` uses `pe.OPTIONAL_HEADER.ImageBase + section.VirtualAddress + section.Misc_VirtualSize` as the base offset.

I'll see if there is something like ImageBase in [debug/pe]